src/electionguard/elgamal.py

```diff
@@ -48,7 +48,7 @@ def decrypt_known_product(self, product: ElementModP) -> int:
         :param product: The known product (blinding factor).
         :return: An exponentially encoded plaintext message.
         """
-        return discrete_log(mult_p(self.data, mult_inv_p(product)))
+        return DiscreteLog().discrete_log(mult_p(self.data, mult_inv_p(product)))
```

Comment:
Doesn't feel super clean, but I don't have something to replace it with besides a function.